person.rb

```diff
@@ -18,3 +31,15 @@ def can_use_services?
     of_age? || @parent_permission
   end
 end
+
+person = Person.new(22, 'maximilianus')
+
+person.correct_name
+
+capitalized_person = CapitalizeDecorator.new(person)
+
+puts capitalized_person.correct_name
+
+capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
+
+puts capitalized_trimmed_person.correct_name
```

Comment:
- [ ] Great job so far 👍🏻 however your logic does not work correctly instead I'm getting `Unknown` in my console

![Screenshot from 2022-11-02 00-12-03](https://user-images.githubusercontent.com/79058364/199352298-deb173e3-19d8-43ee-aece-6fdf18cf685c.png)